### PR TITLE
add underscore.string; listen to search

### PR DESCRIPTION
**Pull request by joonhocho**

### CODE REVIEW

File: kifi-backend/modules/shoebox/angular/src/layout/main/MainCtrl.js [hunk 1]
```diff
@@ -38,5 +38,20 @@ angular.module('kifi.layout.main', [])
     $scope.undo = function () {
       $scope.undoAction = null;
     };
+
+    var throttled = _.throttle(function () {
+      var text = $scope.search.text || '';
+      text = _.str.trim(text);
```
Comment (2is10) [inline kifi-backend/modules/shoebox/angular/src/layout/main/MainCtrl.js:44]: `String.prototype.trim` is [in IE 9 & up](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/Trim#Browser_compatibility). Are we targeting IE 8 with the Angular app?

Comment (joonhocho) [inline kifi-backend/modules/shoebox/angular/src/layout/main/MainCtrl.js:44]: Good question. @andrewconner ?

Comment (andrewconner) [inline kifi-backend/modules/shoebox/angular/src/layout/main/MainCtrl.js:44]: I don't think we've intentionally broken IE 8 support yet (besides minor things, like how the new models will be positioned). I'll talk with product to see what the expectations are... building the site with support is a lot easier than going back and fixing stuff.

Comment (2is10) [inline kifi-backend/modules/shoebox/angular/src/layout/main/MainCtrl.js:44]: http://www.sadtrombone.com/?play=true

**Pull request merged**